### ISSUE TITLE
Retinal periphlebitis

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10934]
Preferred term label:
Retinal periphlebitis

Synonyms
Periphlebitis, Retinal vein inflammation

Definition (free text, please give PubMed ID)
A condition characterized by inflammation of the outer walls of retinal veins, often presenting as vascular sheathing or exudates along the venous walls. Periphlebitis is commonly associated with inflammatory and systemic diseases such as sarcoidosis, multiple sclerosis, or Behçet’s disease. It may lead to retinal vascular occlusion or neovascularization. 

Parent term (use hpo.jax.org/app)
Abnormal retinal vasculature (HP:8046)

Diseases characterized by this term
Sarcoidosis, Behçet’s disease, Multiple sclerosis

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001325
- label: Retinal periphlebitis
- parents: [Abnormal retinal vein morphology (HP:6001165)](https://hpo.jax.org/app/browse/term/HP:6001165)
- pmid: 
- def: A condition characterized by inflammation of the outer walls of retinal veins, often presenting as vascular sheathing or exudates along the venous walls.
- comment: Periphlebitis is commonly associated with inflammatory and systemic diseases such as sarcoidosis, multiple sclerosis, or Behcet disease. It may lead to retinal vascular occlusion or neovascularization.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10934
- synonyms:
